<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>821</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgWidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(243, 243, 243, 255), stop:1 rgba(255, 255, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;Ubuntu&quot;;color:rgb(46, 52, 54)</string>
    </property>
    <property name="text">
     <string>Rented Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 87, 83);
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>619</x>
      <y>0</y>
      <width>351</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(114, 159, 207);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>deliver book ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_edit">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>210</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="book_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>551</width>
      <height>271</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>book id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>author</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>genre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>price</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 87, 83);
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 87, 83);
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
